faq/README.md

```diff
@@ -153,6 +154,11 @@ See more [information about the primary key](/guides/main_concepts/documents.md#
 Your documents upload probably failed.
 To understand what happened, please check this [answer](/faq/README.md#i-did-a-call-to-an-api-route-and-i-only-got-an-updateid-as-a-response-what-does-it-mean).
 
+## Is Killing MeiliSearch process safe?
+
+Killing MeiliSearch even in the middle of a process (ex: adding a batch of document) is **safe**. When you restart the server, it will start the task all over again.
```

Comment:
```suggestion
Killing MeiliSearch is **safe**, even in the middle of a process (ex: adding a batch of documents). When you restart the server, it will start the task from the beginning.
```